### DISCUSSION TITLE
get_artist_albums fetching only first 100 records not all songs

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [sigma67/ytmusicapi#511] [Q&A] [answered]
`resp = ytmusic.get_artist_albums("MPADUCtJe0RYzgPddQXKtWduxz_w", "ggMIegYIARoCAQI%3D")
`
this is the code i am using , any way to get full album songs
hi @sigma67 can you help on this

**Comment by sigma67** [comment, marked as answer]
https://github.com/sigma67/ytmusicapi/issues/461

**Comment by deadhead-pharaoh** [comment]
Does anyone know if this is also the same reason I can't get the "limit" input parameters don't seem to work on methods like get_playlist?  For instance, that method seems to return 200 tracks no matter what limit I pass.

**Reply by sigma67** [reply]
https://ytmusicapi.readthedocs.io/en/stable/faq.html#why-is-ytmusicapi-returning-more-results-than-requested-with-the-limit-parameter
